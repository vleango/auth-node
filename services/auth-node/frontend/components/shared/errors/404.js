import React from 'react';
import { Link } from 'react-router-dom';

export const NotFoundPage = () => (
  <p>404! - <Link to="/">Go home</Link></p>
);

export default NotFoundPage;
