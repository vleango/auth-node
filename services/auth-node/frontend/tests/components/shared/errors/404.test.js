import React from 'react';
import { shallow } from 'enzyme';
import { NotFoundPage } from '../../../../components/shared/errors/404';

let wrapper = shallow(<NotFoundPage />);

describe('Components', () => {
  describe('Shared', () => {
    describe('Errors', () => {
      describe('404', () => {

        it('should correctly render NotFoundPage', () => {
        	expect(wrapper).toMatchSnapshot();
        });

      });

    });
  });
});
