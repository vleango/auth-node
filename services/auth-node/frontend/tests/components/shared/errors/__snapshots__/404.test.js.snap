// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Shared Errors 404 should correctly render NotFoundPage 1`] = `
<p>
  404! - 
  <Link
    replace={false}
    to="/"
  >
    Go home
  </Link>
</p>
`;
